apps/etterlatte-saksbehandling-ui/client/src/shared/icons/statusIcon.tsx

```diff
@@ -10,11 +10,11 @@ export const StatusIcon = (props: { status: VurderingsResultat; noLeftPadding?:
   function hentSymbol() {
     switch (props.status) {
       case VurderingsResultat.OPPFYLT:
-        return <SuccessColored />
+        return <SuccessColored aria-hidden={"true"} /> // Vurder å bruk tittel for å forklare istendenfor å skjule
```

Comment:
Her skjules det ikke men er ikke byttet til tittel?

Reply:
Den skjules nå, men så er spørsmålet om man skal bruke tittel for å forklare ikonet istedenfor å skjule det.

Reply:
Jepp